utils/IconSelection.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+    Email,
+    MailOutline,
+    Send,
+    Facebook,
+    FacebookOutlined,
+    Instagram,
+    Pinterest,
+    Twitter,
+    Reddit,
+    YouTube
+} from '@mui/icons-material'
+
+export const SocialMenuItems = [
+    {
+        icon: <Send className="icon" />,
+        value: 'Send'
+    },
+    {
+        icon: <Email className="icon" />,
+        value: 'Email'
+    },
+    {
+        icon: <MailOutline className="icon" />,
+        value: 'Email Outline'
+    },
+    {
+        icon: <Facebook className="icon" />,
```

Comment:
Why 2 facebook icons?

Reply:
they look different. one is regular one is outlined just to give an extra option